src/ompl/geometric/planners/bitstar/datastructures/src/IntegratedQueue.cpp

```diff
@@ -1578,7 +1578,7 @@ namespace ompl
 
         bool BITstar::IntegratedQueue::isCostBetterThan(const ompl::base::Cost& a, const ompl::base::Cost& b) const
         {
-            return a.value() < b.value();
+            return opt_->isCostBetterThan(a, b);
```

Comment:
So... unfortunately these are not the same thing.

OptimizationObjective::isCostBetterThan() uses a "margin" in it's comparison (i.e, it returns a < b + delta). Since isCostBetterThan is used for more than just a terminating condition (i.e., things are ordered on cost), I've purposefully avoided using the function in the optimization objective.

I know this causes problems for cases where you want to maximize cost, but my plan was to fix it at a later date by proposing a fix to the optimization class. I just haven't had a chance to do so yet.